packages/twenty-front/src/modules/activities/calendar/components/CalendarCurrentEventCursor.tsx

```diff
@@ -22,7 +22,7 @@ type CalendarCurrentEventCursorProps = {
 
 const StyledCurrentEventCursor = styled(motion.div)`
   align-items: center;
-  background-color: ${({ theme }) => theme.font.color.danger};
+  background-color: ${({ theme }) => theme.color.red};
```

Comment:
this should not be necessary if we keep font.color.danger = color.red, see comment below